backend/src/main/java/com/example/backend/Controllers/AccountController.java

```diff
@@ -20,7 +20,7 @@
 @RequiredArgsConstructor
 @Controller
 @RequestMapping
-@CrossOrigin(origins = "http://localhost:3000")
+@CrossOrigin(origins = "*")
```

Comment:
I’m sharing this document because it explains how to use the @CrossOrigin annotation in Spring Boot effectively. In our project, we repeatedly configure CORS in multiple controllers, which can lead to redundancy. This document highlights how to define CORS settings globally, which simplifies maintenance and ensures consistency across the application.

For more details, check it out here:[How to Use @CrossOrigin in Spring Boot](https://medium.com/@dev_RV/what-is-crossorigin-annotation-in-spring-boot-its-purpose-66125e1fc21a#:~:text=How%20to%20Use,can%20do%20it%3A)

Reply:
Good point indeed